### PR TITLE
 Common Footer & #3co | #3ar | #3sp | #3ho

### PR DESCRIPTION
**Pull request by bymayanksingh**
Fix 1: Wrong Twitter link was modified

### CODE REVIEW

**Comment by bymayanksingh** [comment]
Work In Progress: Common Footer

It currently looks like this:
image

Work needed to be done: 

- [x] to correct font awesome icons & add suitable links.

- [x] to change *About PyData* text.

File: src/App.vue [hunk 1]
```diff
@@ -84,3 +131,137 @@ export default {
...
+    .footer-distributed .footer-center,
+    .footer-distributed .footer-right 
+    { 
+        display: block;
+        width: 100%;
+        margin-bottom: 40px;
+        text-align: center;
+    }
+}
+</style>
```
Comment (ramantehlan) [inline src/App.vue]: Add a new line here. :)

**Comment by bymayanksingh** [comment]
@Arsh23 does it look alright?

File: package.json [hunk 2]
```diff
@@ -19,6 +19,9 @@
   },
   "homepage": "https://github.com/pydatadelhi/pydatadelhi.github.io#readme",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.18",
+    "@fortawesome/free-solid-svg-icons": "^5.8.2",
+    "@fortawesome/vue-fontawesome": "^0.1.6",
```
Comment (ramantehlan) [inline package.json]: Where are you using these packages?

**Review by ramantehlan — CHANGES_REQUESTED** [review]
I think the progress so far is good, can you also upload the responsive view of the changes.

File: src/App.vue (hunk 1 above)
Comment (bymayanksingh) [reply src/App.vue]: Sure, will do that.

File: package.json (hunk 2 above)
Comment (bymayanksingh) [reply package.json]: Adding font awesome icon support in Vue, will remove unnecessary packages after I am done with the above mentioned work.

Comment (ramantehlan) [reply package.json]: Check index.html, I guess the support is already added there.

**Comment by bymayanksingh** [comment]
image

@ramantehlan  @codekhal @Arsh23 Please take a look !

**Comment by bymayanksingh** [comment]
It's fully responsive too! 
image

File: package.json (hunk 2 above)
Comment (bymayanksingh) [reply package.json]: Yes, Thanks.

**Comment by bymayanksingh** [comment]
> I think the progress so far is good, can you also upload the responsive view of the changes.

one more (mobile view)
image

**Comment by bymayanksingh** [comment]
Update: 
image

**Comment by bymayanksingh** [comment]
Sponsors Page Update:
image

**Comment by bymayanksingh** [comment]
Update Archive Page: 
image

**Pull request merged**